build_tools/cmake/iree_copts.cmake

```diff
@@ -447,6 +447,10 @@ set(LLVM_TARGETS_TO_BUILD "WebAssembly;X86;ARM;AArch64;RISCV;NVPTX"
     CACHE STRING "" FORCE)
 
 set(LLVM_ENABLE_PROJECTS "mlir" CACHE STRING "" FORCE)
+if(${IREE_BUILD_EXPERIMENTAL_ROCM})
+  set(LLVM_ENABLE_PROJECTS "mlir;lld" CACHE STRING "" FORCE)
```

Comment:
Why do you need this?

Reply:
We'd need that for future patch to assemble and link ISA into code object, I'll remove that for now :)